src/components/userGroups/UserGroup.tsx

```diff
@@ -639,30 +639,32 @@ const CollapsedUserOrGroup: React.FC<CollapsedUserOrGroupProps> = ({
                 </span>
                 <span
                     className="user-list__direction-container flex rotate pointer"
-                    onClick={email_id === 'admin' ? noop : (e) => setCollapsed(not)}
+                    onClick={(email_id === 'admin'||email_id==='system') ? noop : (e) => setCollapsed(not)}
```

Comment:
Extract out this inline function as  individual function, Keep admin and system in constant